Makefile

```diff
@@ -292,7 +292,7 @@ smoke-test: docker smoke-test-run
 
 smoke-test-run: KIND=system_test
 smoke-test-run: internal-build
-	@$(REBAR) as $(KIND) do ct $(ST_CT_DIR) $(ST_CT_FLAGS) --suite=aest_sync_SUITE,aest_commands_SUITE,aest_peers_SUITE
+	@$(REBAR) as $(KIND) do upgrade, ct $(ST_CT_DIR) $(ST_CT_FLAGS) --suite=aest_sync_SUITE,aest_commands_SUITE,aest_peers_SUITE
```

Comment:
Isn't this `upgrade` workaround the lock file?

Reply:
The lock file is only used by the deps in the `default` profile. All additional deps defined in other profiles are not locked. Therefore, this upgrade ensure that they are updated to the defined reference.

Reply:
Interesting tool :)

Reply:
@dincho there is no lock file for `system_test`?

Reply:
And except that the update does not seem to help... Interesting tool indeed :-)